Scripts/FilterByList/FilterByList.yml

```
@@ -1,37 +1,64 @@
 commonfields:
   id: FilterByList
   version: -1
-name: FilterByList
-script: '-'
-type: python
-subtype: python2
-tags:
-- whitelist
-comment: Checks whether the specified item is in a list. The default list is the Demisto
-  Indicators Whitelist.
 enabled: true
+name: FilterByList
 args:
-- name: values
-  default: true
+- default: true
   description: The item to look for in the list.
   isArray: true
-- name: listname
-  description: Name of the list against which to check the value. The default is the Indicators Whitelist.
+  name: values
+  required: false
+  secret: false
+- default: false
   defaultValue: Indicators Whitelist
-- name: ignorecase
-  auto: PREDEFINED
+  description: Name of the list against which to check the value. The default is the
+    Indicators Whitelist.
+  isArray: false
+  name: listname
+  required: false
+  secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: 'No'
+  description: Whether to ignore the case of the item for which you are searching.
+    Default is "No".
+  isArray: false
+  name: ignorecase
+  predefined:
+  - 'yes'
+  - 'no'
+  required: false
+  secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: 'No'
+  description: Whether to match the exact item in the list, ot look for any string
```

No and Yes are not intuitive options 
```suggestion
  description: Whether to match the exact item in the list, or look for any string
```